capi/src/write/filters-by-route.ts

```diff
@@ -16,9 +16,13 @@ export async function filtersByRoute(
             ? ((): string | undefined => {
                 const filterKey = page?.filterKey as string | undefined;
                 return filterKey
-                  ? JSON.stringify({
-                      [filterKey]: ctx.config.filters[filterKey],
-                    })
+                  ? page.route.includes("/sdk")
```

Comment:
The chained ternary operator here can be cleaned up. Maybe move these into functions.